display_list/display_list_builder.h

```diff
@@ -342,7 +342,12 @@ class DisplayListBuilder final : public virtual Dispatcher,
 
   sk_sp<DisplayList> Build();
 
+  unsigned save_count() const { return save_count_; }
```

Comment:
This appears to be a new statistic for the total number of save calls added to the stream. In order to distinguish itself from `getSaveCount()` it should probably be called either `total_save_count_` or `save_records_count_`?

If this is just for testing, then I think some good tests using the verbose DL comparison testing facility would be more targeted, but I do acknowledge that this adds a quick extra check to all of our existing unit tests. Were any of those additional checks of the "save_count()" informative in terms of noticing optimized save/restores?